src/uwtools/tests/config/formats/test_base.py

```diff
@@ -213,16 +213,25 @@ def test_dereference(tmp_path):
   c: !int '{{ N | int + 11 }}'
 d: '{{ X }}'
 e:
-  - !int '42'
-  - !float '3.14'
-  - !datetime '{{ D }}'
   - !bool "False"
+  - !datetime '{{ D }}'
+  - !float '3.14'
+  - !int '42'
+  - !dict "{ b0: 0, b1: 1, b2: 2,}"
+  - !dict "[ ['c0',0], ['c1',1], ['c2',2], ]"
```

Comment:
Alphabetical ordering to match other sections. Unless there's more of a reason it has to be in this order. In that case, I'm curious to why that is needed.
```suggestion
  - !dict "{ b0: 0, b1: 1, b2: 2,}"
  - !dict "[ ['c0',0], ['c1',1], ['c2',2], ]"
    - !int '42'
```